src/lib/components/workflow/workflow-error.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+  import { page } from '$app/stores';
+
+  import Link from '$lib/holocene/link.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { workflowsSearchParams } from '$lib/stores/workflows';
+  import type { NetworkError } from '$lib/types/global';
+  import { routeForWorkflows } from '$lib/utilities/route-for';
+  import Logo from '$lib/vendor/logo-dark.svg';
+
+  export let error: NetworkError;
+
+  $: ({ namespace } = $page.params);
+
+  $: title =
+    error.statusCode === 404
+      ? translate('workflows.workflow-404-title')
+      : translate('workflows.workflow-error-title');
+</script>
+
+<header class="mb-4 flex flex-col gap-1">
+  <div class="mb-4 block">
+    <Link
+      href={`${routeForWorkflows({
+        namespace,
+      })}?${$workflowsSearchParams}`}
+      data-testid="back-to-workflows"
+      icon="chevron-left"
+    >
+      {translate('workflows.back-to-workflows')}
+    </Link>
+  </div>
+  <h1
+    data-testid="workflow-error-heading"
+    class="overflow-hidden text-3xl font-medium"
+  >
+    {error.statusText}
```

Comment:
What do you think about making this a little more similar to the existing `Error` page and have the status text centered with the other info. ⬇️ 

![Screenshot 2024-01-24 at 11 32 11 AM](https://github.com/temporalio/ui/assets/15069288/66dd52dd-6d7b-45b3-83be-4ac8dc4891c1)

Reply:
👍 updated